core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1279,24 +1279,34 @@ public String toString() {
       new Builder(Name.MASTER_EMBEDDED_JOURNAL_ELECTION_TIMEOUT)
           .setDescription(
               "The election timeout for the embedded journal. When this period elapses without a "
-                  + "master receiving any messages, the master will attempt to become the primary.")
-          .setDefaultValue("5s")
+                  + "master receiving any messages, the master will attempt to become the primary."
+                  + "Election timeout will be waited initially when the cluster is forming. "
+                  + "So larger values for election timeout will cause longer start-up time. "
+                  + "Smaller values might introduce instability to leadership.")
+          .setDefaultValue("10s")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
           .build();
   public static final PropertyKey MASTER_EMBEDDED_JOURNAL_HEARTBEAT_INTERVAL =
       new Builder(Name.MASTER_EMBEDDED_JOURNAL_HEARTBEAT_INTERVAL)
           .setDescription(
               "The period between sending heartbeats from the embedded journal primary to "
                   + "followers. This should be less than half of the election timeout "
-                  + "(alluxio.master.embedded.journal.election.timeout).")
-          .setDefaultValue("1s")
+                  + "(alluxio.master.embedded.journal.election.timeout), because the election "
```

Comment:
NIT: instead of hardcoding it, can we just use `String.format()` and programmatically include the actual name?